Sources/SwerverCore/Action/HeadAction.swift

```diff
@@ -1,7 +1,7 @@
 public class HeadAction: HttpAction {
 
     private let responseBuilder = ResponseBuilder()
-
+    
```

Comment:
Remove trailing whitespace. In fact, most editors have a feature to automatically trim whitespace on save.

Reply:
fixed